apps/web/src/services/project/composables/use-project-tree.ts

```diff
@@ -42,6 +39,7 @@ const fetchProjectGroupChildMap = async (groups: ProjectGroupModel[]): Promise<R
     const res = {};
 
     const { results: allChildren }: ProjectGroupListResponse = await SpaceConnector.clientV2.identity.projectGroup.list({
```

Comment:
```suggestion
    const { results: allChildren }: ProjectGroupListResponse = await SpaceConnector.clientV2.identity.projectGroup.list<ProjectGroupListRequestParams, ProjectGroupModel>({
```
We can use generic for Api type